### ISSUE TITLE
Autocomplete does not work correctly

### ISSUE THREAD

**Issue by maciejzys** [eranif/codelite#2456]
Hello,
Just recently I have started to use your IDE for c++ development. I am not getting a suggestions from autocomplete utility when I would like to use an object method (for example a vector).

I have had to check online the correct names of the methods. Note that when I am typing, suggestions of the variable names are shown.

Could you please advise me what to do?

All the best,

Maciej

```
#include <iostream>
#include <vector>

using namespace std;

int main()
{
    vector <int> vector1;
    vector <int> vector2;
    
    vector1.push_back(10);
    vector1.push_back(20);

    cout << "vector1: " << vector1.at(0);
    cout << "\nvector1: " << vector1.at(0);
    cout << "\nvector length: " << vector1.size();
    
    
    return 0;
}

```

**Comment by eranif** [comment]
This is mostly questions for the forum. And not really bug. In general you should configure language server. You did not mention which OS you are using, so I will assume Linux/Ubuntu.
https://wiki.codelite.org/pmwiki.php/Main/LanguageServer

**Comment by eranif** [comment]
this is now fixed with the release of 14.0.1
